# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1333410295 14400
#      Mon Apr 02 19:44:55 2012 -0400
# Node ID 273fdfcdbb85355866befb25b5d26d41ed6c09dd
# Parent  34a9fdaa6ea32fab08d1a39b89d716be65474971
Remove "tetramesh.m" from the list of unimplemented functions.

* scripts/help/unimplemented.m (missing_functions): Remove "tetramesh".

diff --git a/scripts/help/unimplemented.m b/scripts/help/unimplemented.m
--- a/scripts/help/unimplemented.m
+++ b/scripts/help/unimplemented.m
@@ -355,17 +355,16 @@ function list = missing_functions ()
   "streamtube",
   "strings",
   "subvolume",
   "superclasses",
   "support",
   "surf2patch",
   "symmlq",
   "syntax",
-  "tetramesh",
   "texlabel",
   "textwrap",
   "tfqmr",
   "timer",
   "timerfind",
   "timerfindall",
   "timeseries",
   "toolboxdir",
